sdk/python2/api.c

```diff
@@ -12,7 +12,6 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#define Py_LIMITED_API
```

Comment:
what's the reason for this?

Reply:
Fyi, https://docs.python.org/3/c-api/stable.html

This flag has no effect in python2. For python3, it guarantees some backward compatibility in the ABI, but we don't have evidences that it is needed. I encountered several errors while trying to create the bindings for py3 that were solved by removing this flag. There are several portions of object.h that are disabled in compilation time if the flag is present. All things considered, I believe we don't need this flag.